### ISSUE TITLE
REFPROP does not work with MATLAB

### ISSUE THREAD

**Issue by ibell** [ibell/coolprop#3]
Works fine in other languages, but not in REFPROP:

```
>> Props('D','T',300,'Q',0,'REFPROP-R134a')
??? Error using ==> Props
CoolProp Error: CoolProp error: Your refrigerant [REFPROP-R134a] is from REFPROP, but CoolProp does not support REFPROP on this platform,
yet.
```

**Comment by jowr** [comment]
Could this be related to the EES error?

**Comment by ibell** [comment]
I have fixed one of the EES errors, this seems to be something different.
I have no idea why MATLAB is not working with REFPROP.  Do you have MATLAB
on your computer?  Could you give this a try?

**Comment by jowr** [comment]
I meant issue number 2, not 1. Sure it is not related?

A fresh installation of the MATLAB wrapper seems to work fine. I have a
32bit Linux machine.

I cannot test the Windows part, no Matlab on Windows...

**Comment by ibell** [comment]
Ah - maybe its the old 32-bit/64-bit issue again.  I tested using MATLAB
64-bit, that could be the problem.  I think REFPROP is 32-bit, right?

**Comment by jowr** [comment]
Yes and no. The DLL comes in 32bit, but you can also get a 64bit version from NIST. On other platforms (Linux, OSX) refprop is compiled for the specific architecture (see librefprop.so). Hence, we have 64bit if the machine is a running a 64bit OS.

**Comment by ibell** [comment]
Ok, got it - definitively a problem with 32-bit REFPROP not being able to
be called from 64-bit MATLAB.  How do we test this at run time?

**Comment by jowr** [comment]
How about something like this
http://msdn.microsoft.com/en-us/library/aa364819%28VS.85%29.aspx or this
http://stackoverflow.com/a/12338526

**Comment by ibell** [comment]
That should do the trick for figuring out what bitness was used to build,
but I am not sure that tells us about the bitness of the DLL - I'll have to
dig a bit

**Comment by ibell** [comment]
Ok, what we need to do is to try to load REFPRP64.DLL if we are 64-bit compiled, REFPROP.dll otherwise.  Not sure how this should be handled

**Comment by ibell** [comment]
Fixed in revision 0623efc12909699d20f8d798a9e658931226c76c
